cfme/base/__init__.py

```diff
@@ -259,11 +261,18 @@ def collect_logs(self):
 
     @property
     def region(self):
-        zone_res = self.appliance.rest_api.collections.zones.find_by(id=self.id)
-        zone, = zone_res
-        zone.reload(attributes=['region_number'])
-        region_obj = self.appliance.collections.regions.instantiate(number=zone.region_number)
-        return region_obj
+        possible_parent = parent_of_type(self, Region)
```

Comment:
I think a cached_property would be more appropriate here, even though this is something that likely won't need to be deleted.

No need to take `_region` as an attribute, just return `parent_of_type()` result, or the rest API lookup if that's empty.  Decorate the method with `@cached_property` and there's no need to track `self._region`